src/dev/pages/stack/stack.ejs

```diff
@@ -53,20 +155,40 @@
         <p slot="start">Complex form</p>
       </forge-toolbar>
       <form style="padding: 16px;">
-        <forge-stack gap="16" stretch>
-          <forge-stack inline wrap>
-            <forge-text-field style="--forge-stack-stretch: 2">
-              <input type="text" id="input-text-01" />
-              <label for="input-text-01" slot="label">Text field</label>
-            </forge-text-field>
-            <forge-text-field style="--forge-stack-stretch: 4">
-              <input type="text" id="input-text-01" />
-              <label for="input-text-01" slot="label">Text field</label>
-            </forge-text-field>
-            <forge-text-field style="--forge-stack-stretch: 6">
+        <forge-stack>
+          <forge-stack gap="16">
```

Comment:
We should remove the `gap` attribute when it matches the default value. Just makes it easier when we share these demos out so people don't think they _need_ it.